AP-protocol.md

```diff
@@ -130,7 +133,8 @@ For every response or acknowledgment the initiator receives, it updates its unde
 
 The flood can terminate when:
 
--
+- Timeout after a certain period, depending on the number of nodes and on the maximum timeout in communication channels
+- A drone crashes in the network discovery phase. As soon as the initiator knows it, it starts a new flooding with a new flood_id, and it starts ignoring QueryResult messages about the previous flood_id.
```

Comment:
We think this is a Client behaviour. The termination condition should refer to the condition that triggers the send of the QueryResult message and for us the TTL is enough.